src/main/resources/data/minecolonies/researches/civilian/1_higherlearning.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "higherlearning",
+  "name": "Higher Learning",
+  "branch": "civilian",
+  "exclusiveChildResearch": true,
+  "requiredUniversityLevel": 1,
+  "requiredItems":
+  [
+  {
+    "itemName": "minecraft:book",
+    "itemCount": 3
+  }
+  ],
+  "requiredBuildings":
+  [
+  {
+    "building": "citizen",
+    "buildingLevel": 3
+  }
+  ],
+  "effects":
+  [
+  {
+    "unlockBuilding": "School",
```

Comment:
So, if we know the building here, why do we have all the buildings/blocks with overrides for validating individual researches? We should be able to have a single baseclass implementation that checks if there is an unlock for the current building that needs to be unlocked. As a modpack/datapack author, I may not want to have researches in place for particular buildings, I may be gating on a questbook.